pkg/meta/store_test.go

```diff
@@ -26,6 +26,7 @@ func (d *Demo2) Key() string {
 var s *Store
 
 var cfg = Config{
+	Driver:  "local",
```

Comment:
Could you add unit test for store driver "boltdb"?

Reply:
ok